src/pages/ReportView/WorkloadInventory/WorkloadInventory.tsx

```diff
@@ -1,27 +1,487 @@
 }
 
 interface DispatchToProps {
-    fetchReport: (reportId: number) => void;
+    fetchReportWorkloadInventory: (
+        reportId: number,
+        page: number,
+        perPage: number,
+        orderBy: string | undefined,
+        orderDirection: 'asc' | 'desc' | undefined
+    ) => any;
+    fetchReportWorkloadInventoryCSV:(reportId: number) => any;
 }
 
 interface Props extends StateToProps, DispatchToProps {
+    reportId: number;
 };
 
+interface Row extends IRow {
+    isOpen?: boolean;
+}
+
+interface Column extends ICell {
+    key: string;
+    cellFormatters?: any;
+    transforms?: any;
+}
+
 interface State {
-    reportId: number;
+    page: number;
+    perPage: number;
+    columns: Column[];
+    rows: Row[];
+    sortBy: ISortBy;
 };
 
 class WorkloadInventory extends React.Component<Props, State> {
 
+    public changePage = debounce(() => {
+        this.refreshData();
+    }, 800);
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            page: 1,
+            perPage: 10,
+            columns: [
+                {
+                    title: 'Provider',
+                    key: 'provider',
+                    props: {
+                        className: 'vertical-align-middle'
+                    },
+                    cellFormatters: [ expandable ]
+                },
+                {
+                    title: 'Datacenter',
+                    key: 'datacenter',
+                    props: {
+                        className: 'vertical-align-middle'
+                    }
+                },
+                {
+                    title: 'Cluster',
+                    key: 'cluster',
+                    props: {
+                        className: 'vertical-align-middle'
+                    }
+                },
+                {
+                    title: 'VMware',
+                    key: 'vmName',
+                    props: {
+                        className: 'vertical-align-middle'
+                    },
+                    transforms: [ sortable ]
+                },
+                {
+                    title: 'Workload',
+                    key: 'workload',
+                    props: {
+                        className: 'vertical-align-middle'
+                    }
+                },
+                {
+                    title: 'OS type',
+                    key: 'osName',
+                    props: {
+                        className: 'vertical-align-middle'
+                    },
+                    transforms: [ sortable ]
+                },
+                {
+                    title: 'Efford',
+                    key: 'complexity',
+                    props: {
+                        className: 'vertical-align-middle'
+                    },
+                    transforms: [ sortable ]
+                },
+                {
+                    title: 'Recomended targets',
```

Comment:
```suggestion
                    title: 'Recommended targets',
```
Here, in the mock ups, `Recommended` and `targets` are in 2 lines, not in the same one, can we do it?

Reply:
I've tried using Patternfly styles but I couldn't do it. I really like the way it is currently implemented (just one line); let me ask this to the UX team

Reply:
Ok, great, thanks.
Let's wait for their opinion to understand if we can keep it as it is or we need to change it

Reply:
I don't have a problem with a one line. It seems to be ok.

Reply:
Thanks @doruskova, let's keep it as it is, no further change required :+1: